app/database/models/social_sign_in.py

```diff
@@ -0,0 +1,46 @@
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+
+class SocialSignInModel(db.Model):
+    """Data model representation of social sign in of a user.
+
+    Attributes:
+        user_id: user_id, to identify the user in user model
+        social_sign_in_type: social sign in type (apple, google)
+        id_token: id_token sent by the social sign in provider
+        associated_email: email of the user associated with the social sign in provider
+        full_name: full name of the user associated with the social sign in provider
+    """
+
+    # Specifying database table used for SocialSignInModel
+    __tablename__ = "social_sign_in"
+    __table_args__ = {"extend_existing": True}
+
+    # data properties
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"))
+    social_sign_in_type = db.Column(db.String(20))
+    id_token = db.Column(db.String(100), primary_key=True)
+    associated_email = db.Column(db.String(50))
+    full_name = db.Column(db.String(50))
+
+    def __init__(self, user_id, sign_in_type, id_token, email, name):
```

Comment:
Since you are using type hints in the other functions of this file, why not use here :)

Reply:
I'm not clear about this point.